app/models/person.rb

```diff
@@ -23,6 +23,8 @@ class Person < ApplicationRecord
   accepts_nested_attributes_for :location
 
   validate :preferred_contact_method_present!
+  validates :email, email: true
+  validates :email_2, email: true
```

Comment:
I like the `validates_with` methods a little better, because they use a class name. If the class name `EmailValidator` is here in this validation line, someone reading this code will have to do less thinking to guess where the logic exists. I'm sure there's other ways to write this that accomplish the same thing. What I care about is that these `email: true` arguments feel like too much "Rails magic" too me

Reply:
Done :+1: